docs/content/docs/deployment/overview.md

```diff
@@ -158,7 +158,12 @@ When deploying Flink, there are often multiple options available for each buildi
 Once a job has reached a globally terminal state of either finished, failed or cancelled, the
 external component resources associated with the job are then cleaned up. In the event of a
 failure when cleaning up a resource, Flink will attempt to retry the cleanup. You can
-[configure]({{< ref "docs/deployment/config#retryable-cleanup" >}}) the retry strategy used.
+[configure]({{< ref "docs/deployment/config#retryable-cleanup" >}}) the retry strategy used. 
+Reaching the maximum number of retries without succeeding will leave the job in a dirty state. 
```

Comment:
Do we have any documentation that explains the consequences of being in a dirty state?

Reply:
I updated the documentation under `Deployment / High Availability / Overview / JobResultStore` to also cover dirty entries with references to the folders.